clients/cobol-lsp-vscode-extension/src/services/ProcessorGroupsLoader.ts

```diff
@@ -20,7 +20,17 @@ import { TextDecoder } from "util";
 const PG_FOLDER = ".cobolplugin";
 const PGR_PGM_FILE = "pgm_conf.json";
 const PG_PROC_FILE = "proc_grps.json";
+const DEFAULT_PROCESSOR_GROUP_CONFIG = "[]";
+const DEFAULT_PROCESSOR_CONFIG = '{ "pgms": [] }';
```

Comment:
```suggestion
const DEFAULT_PROGRAM_CONFIG = '{ "pgms": [] }';
```

Reply:
I think it would make sense to just have these as already processed objects instead of strings that need to be parsed.